src/docs/development/tools/devtools/performance.md

```diff
@@ -3,6 +3,13 @@ title: Using the Performance view
 description: Learn how to use the DevTools performance view.
 ---
 
+{{site.alert.note}}
+  The performance view works with mobile apps only.
+  Use Chrome DevTools to [analyze
```

Comment:
You also need to run the app with `--profile` for the Chrome profile to make any sense.